tests/phpunit/test-main.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Pantheon MU Plugin Tests
+ * 
+ * @package pantheon
+ */
+
+/**
+ * Main Mu Plugin Test Case
+ */
+class Test_Main extends WP_UnitTestCase {
+	/**
+	 * Test that the mu-plugin is loaded.
+	 */
+	public function test_mu_plugin_constants() {
+		$this->assertTrue( defined( 'PANTHEON_MU_PLUGIN_VERSION' ) );
+		$this->assertEquals( '1.2.1', PANTHEON_MU_PLUGIN_VERSION );
```

Comment:
this seems tedious to keep in sync. maybe just assertTrue above?

Reply:
my thought is that this should remind us to bump the version if we forgot (because we'll need to bump it in the tests at least). there's perhaps a better mechanism for ensuring this, but, as above, we don't currently have a release workflow for the mu-plugin documented or standardized.

Reply:
If we're cutting releases on merge to main (which this does, I think insofar as updatinator pulls from main), this repo should be on autotag and the version bumps entirely automated.